Source/WebCore/html/HTMLElement.idl

```diff
@@ -48,9 +48,14 @@
 
     [EnabledBySetting=FormAssociatedCustomElementsEnabled] ElementInternals attachInternals();
 
-    // Experimental: https://github.com/WICG/inert/blob/master/README.md
     [CEReactions, Reflect, EnabledBySetting=InertAttributeEnabled] attribute boolean inert;
 
+    // The popover API
+    [EnabledBySetting=PopoverAttributeEnabled] undefined showPopover();
+    [EnabledBySetting=PopoverAttributeEnabled] undefined hidePopover();
+    [EnabledBySetting=PopoverAttributeEnabled] undefined togglePopover(optional boolean force);
+    [CEReactions, EnabledBySetting=PopoverAttributeEnabled] attribute [AtomString] DOMString? popover;
```

Comment:
We should add a custom element reaction test. Can be done in a separate PR though.

Reply:
I'll create a separate PR to add some to LayoutTests/imported/w3c/web-platform-tests/custom-elements/reactions/HTMLElement.html